setup.cfg

```diff
@@ -21,3 +21,5 @@ install_requires=
     oxenmq
     oxenc
     pyonionreq
+    tomil
```

Comment:
What is this?  I can't find any such python package.

Reply:
typo, it is tomli